ClassLibrary/ClassLibrary/DepartmentReaders.cs

```diff
@@ -81,5 +92,36 @@ public void BorrowBook(Reader reader,Book book)
 
             reader.AddBookInCard(book);
         }
+
+        public void ReturnBook(int IDReader, String NameBook)
```

Comment:
I understand why this method has the same parameters as `BorrowBook`, but it can't be the same.
imagine real life situations: 
1. reader comes to the librarian and says "I need War and Peace" - all librarian has to fulfill that request is reader's card with ID and a book name.
2. reader comes to the librarian and says "I want to return this War and Peace book" - librarian doesn't need to check look for a book by its name - the book is already there, it has a tag on it.

so there should be just book id instead of book name.

Reply:
not done yet

Reply:
I have corrected, but it may be like this. The librarian will not check by code (if there is no scanner), if it contains 10 digits, it is easier for him to check by name. And I doubt that you will take 50 books

Reply:
but the librarian have to make sure that the same book has returned

Reply:
не понял это был ответ или это было замечнаие добавить еще доп. проверку